sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1042,3 +1045,20 @@ def __sklearn_is_fitted__(self):
 
     check_fit_check_is_fitted("estimator", Estimator(behavior="method"))
     check_fit_check_is_fitted("estimator", Estimator(behavior="attribute"))
+
+
+def test_non_deterministic_estimator_skip_tests():
+    # check estimators with non_deterministic tag set to True
+    # will skip certain tests
```

Comment:
Also put the issue number here in the comment for future reference, otherwise LGTM!

Reply:
Hi @adrinjalali , I've added the issue number in the comments as suggested :)